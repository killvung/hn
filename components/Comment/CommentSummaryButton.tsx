import React from 'react';

const CommentSummaryButton: React.FC = () => {
    return (
        <summary className="govuk-details__summary">
            <span className="govuk-details__summary-text" />
        </summary>
    );
}

export default CommentSummaryButton;
