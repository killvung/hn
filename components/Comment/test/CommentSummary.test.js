import React from 'react';
import renderer from 'react-test-renderer';
import CommentSummaryButton from '../CommentSummaryButton';

test('Comment - when toggle is true (on), comment section expanded, label changes to collasp', () => {
    const tree = renderer
        .create(<CommentSummaryButton toggle={true} />)
        .toJSON();
    console.log(tree);
    expect(tree).toMatchSnapshot();
});

test('Comment - when toggle is false (off), comment section collasped, label changes to expand', () => {
    const tree = renderer
        .create(<CommentSummaryButton toggle={false} />)
        .toJSON();
        console.log(tree);
    expect(tree).toMatchSnapshot();

});