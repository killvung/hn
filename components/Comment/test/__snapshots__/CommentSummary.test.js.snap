// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment - when toggle is false (off), comment section collasped, label changes to expand 1`] = `
<summary
  className="govuk-details__summary"
>
  <span
    className="govuk-details__summary-text"
  />
</summary>
`;

exports[`Comment - when toggle is true (on), comment section expanded, label changes to collasp 1`] = `
<summary
  className="govuk-details__summary"
>
  <span
    className="govuk-details__summary-text"
  />
</summary>
`;
